include/podio/ObjectID.h

```diff
@@ -9,7 +11,7 @@ class ObjectID {
   /// index of object in collection
   int index;
   /// ID of the collection
-  int collectionID;
+  uint64_t collectionID;
```

Comment:
Do these still work nicely with the `untracked` and `invalid` constants below?

Reply:
Yes. Because we check only on the ObjectID and there the index is more than enough